src/RSCoin/Bank/Server.hs

```diff
@@ -301,3 +309,22 @@ serveGetExplorers sk st =
     do curPeriod <- query st GetPeriodId
        map fst . filter ((== curPeriod) . snd) <$>
            query st GetExplorersAndPeriods
+
+serveAddMintetteUsingPermit
+     :: C.WorkMode m
+     => State -> C.SecretKey -> PublicKey -> C.WithSignature (String, Int) -> ServerTESigned m ()
+serveAddMintetteUsingPermit st bankSK mintettePK signed = do
+  toServerSigned bankSK $ do
+    permittedMintettes <- query st GetPermittedMintettes
+    unless (Set.member mintettePK permittedMintettes) $
+      throwM $ C.BadRequest "Mintette public key doesn't exist"
```

Comment:
`Key doesn't exist` sounds strange. We don't know if it exists or not (I don't even know what does key existence mean). Actual reason is that `key doesn't have permission` or something like that.